docs/build-the-docs.md

```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
     -v $(pwd):$(pwd) \
     -w $(pwd) \
     antora/antora:1.0.1 \
-    generate site.yml
+    generate \
+        --clean \
+        --pull \
+        --quiet \
+        --silent \
+        --ui-bundle-url <url-or-path-to-ui-bundle.zip> \
+        --url <site.url> \
+        site.yml
 ```
+This build command configuration:
 
-This command starts up the Antora Docker container, removes the existing documentation, and then runs Antora's generate command, an regenerates the documentation.
-If all goes well, you will _not_ see any console output.
+- Suppresses all output (`quiet` and `silent`)
+- Removes the output directory before publishing the site (`clean`)
+- Downloads updates from remote resources (`pull`)
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
+
+`ui-bundle` is required, as this isn't specified in the production Playbook file.
```

Comment:
``ui-bundle`` --> ``ui-bundle.zip``

Reply:
Why didnt you add the note like you did for site.prod.yml that ui-bundlie can be found in root, is there a specific reason?